modules/mobile-agents/android/client/plugins/IDPProxy/src/org/wso2/mdm/agent/proxy/IdentityProxy.java

```diff
@@ -66,7 +79,7 @@ public static synchronized IdentityProxy getInstance() {
 		return identityProxy;
 	}
 
-	//initial point of contact
+	// initial point of contact
 	public void init(String clientID, String clientSecret, String username, String password,
```

Comment:
Let's please introduce a bean here as well to wrap all these attributes (If possible, please re-use the bean suggested in one of my previous comments).